### DISCUSSION TITLE
How  to change the provided segmentation labels of SwinUNETR

### DISCUSSION THREAD

**Discussion by laizhui** [Project-MONAI/MONAI#7545] [Q&A] [answered]
I want to train a SwinUNETR model on my own MRI dataset, using the code and model.pt file given at:
https://github.com/Project-MONAI/research-contributions/tree/main/SwinUNETR/BRATS21
For example, the provided segmentation labels have values of 1 for NCR, 2 for ED, 4 for ET, and 0 for everything else. If I want to use 3 to represent ET,what should I do with the code?
Thank you very much!

**Comment by KumoLiu** [comment, marked as answer]
Hi @laizhui, thanks for your interest here.
Instead of modifying the weight and model, adjusting your data should be faster and simpler.
https://github.com/Project-MONAI/MONAI/blob/95f69dea3d2ff9fb3d0695d922213aefaf5f0c39/monai/transforms/utility/array.py#L1166

Hope it helps, thanks.

**Reply by laizhui** [reply]
I try to modify my dataset directly, it works.
Thank you very much!
